lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/restart/DFSEvaluatorLogOverwriteReaderWriter.java

```diff
@@ -82,7 +82,7 @@ public synchronized void writeToEvaluatorLog(final String formattedEntry) throws
           inputStream = newEntryInputStream;
         }
 
-        IOUtils.copyBytes(inputStream, outputStream, 4096, true);
+        IOUtils.copyBytes(inputStream, outputStream, 4096, false);
       } finally {
```

Comment:
We could also move the `inputStream` into an inner "try-with-resources" block to make it even cleaner.